gebru/roster.json

```diff
@@ -34,9 +34,13 @@
       "github": "akennedy205"
     },
     {
-      "name": "David Quigley",
+      "name": "Amarachi Umeloh",
```

Comment:
Hi, please can you create a new object with your name and github name at the bottom the of the roster. Have a look at the other examples on the page and remember to include the comma.